src/components/webcams/streamers/WebrtcCameraStreamer.vue

```diff
@@ -243,6 +249,10 @@ export default class WebrtcCameraStreamer extends Mixins(BaseMixin, WebcamMixin)
 </script>
 
 <style scoped>
+.position-relative {
```

Comment:
there is an position relative `p-rel` css class in in the general page.css. so this would be a duplicate